tools/analysis_tools/report_accuracy.py

```diff
@@ -15,17 +15,13 @@ def parse_args():
         '--preds',
         nargs='+',
         help='list of predict result',
-        default=['demo/fuse/rgb.pkl', 'demo/fuse/flow.pkl'])
```

Comment:
Are these modifications necessary for this PR?

Reply:
I replaced these two files with the dumped results of our new STGCN model and updated the corresponding tutorial. The modifications are necessary since I refer this tutorial when reporting the multi-stream results in this [STGCN_README.md](https://github.com/open-mmlab/mmaction2/blob/cc4d0db85826bed8b5ea93cab4399c31da7aa39f/configs/skeleton/stgcn/README.md).